### DISCUSSION TITLE
Add IP address button on device interface gone? Bug?

### DISCUSSION THREAD

**Discussion by tfboy** [netbox-community/netbox#10375] [Q&A]
Not sure if this is a desired feature or not, but since 3.3.1 and 3.3.2, I no longer have the green + sign next to an interface to add / assign IP address. This is a shame as I now have to go into the interface to find the "+ Add IP Address" button at the bottom.

Is this removal deliberate or is it a bug? I can't see anything in the recent release notes covering this issue.

**Comment by tfboy** [comment]
Apologies, I've just seen the blue + with the dropdown that gives the option. I feel stupid - maybe it's a 3.3.2 thing (I just upgraded from 3.3.1).

**Reply by candlerb** [reply]
It was earlier than that.  Here it is on v3.2.9:

![image](https://user-images.githubusercontent.com/44789/190503287-ba6894c7-db58-41bd-8c7e-1e5d533db96a.png)
